flask_restplus/api.py

```diff
@@ -576,27 +576,34 @@ def handle_error(self, e):
         '''
         got_request_exception.send(current_app._get_current_object(), exception=e)
 
+        include_message_in_error = current_app.config.get("ERROR_INCLUDE_MESSAGE", True)
```

Comment:
I would rename this into `include_message_in_response`

Reply:
Renamed.